giving-a-presentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Giving a Presentation
+sidebar_link: false
+---
+
+Boston Python's presentation events are a place for members of our community
+to share what they've learned about making python programs. Giving a
+presentation to our audience is a way to give back to the community, and it
+helps you understand the subject matter better - it's a win all around!
+
+Presenters do not need to be "experts", they do not need to have contributed to
+the python language, created a library, or written a book. Anyone who has worked
+with the language and learned something interesting from doing so has something
+to share, and our community wants to hear you share it.
+
+### How do I know if I have something to say?
+If you're not sure whether you've got a Boston Python presentation in you, take
+this quiz:
+
+- Have you learned something about the python language?
```

Comment:
Thank you for putting together this guide. Nice to find out that folks are welcome to give presentations.

I am not sure if you are looking for feedback from one of the core maintainers, but one small nitpick if you allow:

the statement "Have you learned something about the python language?" to me is asking about whether someone discovered an artifact of the language, whereas your prior statement "Boston Python's presentation events are a place for members of our community to share what they've learned about making python programs" is more general, in that it is a bout "learning something when building python programs" ...

Reply:
Thanks for that, that's exactly the sort of feedback I was looking for!

Would it be better if we asked "Have you learned something about writing programs in python?"
(alternatively, is this "quiz" idea just too cute? we could probably drop it without doing any harm)